### DISCUSSION TITLE
metadata: anatomy

### DISCUSSION THREAD

**Discussion by bendichter** [dandi/helpdesk#20] [Q&A] [answered]
How do I add an UBERON ID for an anatomical structure?

![image](https://user-images.githubusercontent.com/844306/126202385-30f12c2d-de78-4315-b364-7e935194bba7.png)

**Comment by satra** [comment, marked as answer]
this is due to a VJSF issue: https://codepen.io/satra-the-reactor/pen/PomjgYK?editors=1010 in this particular case, we may update the schema to just use the target class instead of uri + a compact uri class.
